### ISSUE TITLE
Establish a Word template gallery?

### ISSUE THREAD

**Issue by ajfriend** [quarto-dev/quarto-cli#1432]
I'll often need to output documents using a [Word template](https://quarto.org/docs/output-formats/ms-word-templates.html) (where the ultimate target is actually Google Docs instead of Microsoft word).

With my limited design skills, I've messed around with the default template a bit in the past with mixed results.

I think there would be value in establishing a centralized repo or gallery for community-created templates, and it would probably get more traction if it was "official" in some sense, perhaps linked to the documentation here: https://quarto.org/docs/output-formats/ms-word-templates.html

Thoughts?

**Comment by jjallaire** [comment]
Agreed this would be a great service! If you started something like this we could link to it from the docs and then perhaps at some point if you were open to it "make it official" w/ a quarto-dev hosted repository. We'd do this now but are totally buried w/ other stuff.

Note that you could consider having these bundled as format extensions (https://quarto.org/docs/extensions/formats.html). That would enable you to both provide a template and any appropriate option defaults, and would allow for usage w/o too much boilerplate..

**Comment by ajfriend** [comment]
I wasn't aware of extensions. Very cool!

I'll take a look at this again (including considering extensions) at some point in the future, but I also wouldn't be able to quote any specific timeline. If anyone else stumbles over this and feels inspired to carry the torch... 😄
